code/modules/mining/machine_redemption.dm

```diff
@@ -59,6 +59,11 @@
 			var/obj/item/stack/sheet/s = new processed_sheet(src,0)
 			s.amount = 0
 			stack_list[processed_sheet] = s
+			if(s.name != "glass" && s.name != "metal") // no need to announce metal/glass
+				var/msg = "[capitalize(s.name)] sheets now available at Cargo Bay."
+				for(var/obj/machinery/requests_console/D in allConsoles)
+					if(D.department == "Science" || D.department == "Robotics" || D.department == "Research Director's Desk" || D.department == "Chemistry" || D.department == "Bar" )
```

Comment:
Geez, this could really use a var or something you only have to check once per console.